### PR TITLE
feat(General): multiple style changes in components and color changes in Movistar skin 

### PR DESCRIPTION
**Pull request by atabel**
https://jira.tid.es/browse/WEB-558

### CODE REVIEW

File: src/skins/types.tsx [hunk 1]
```diff
@@ -94,7 +94,8 @@ export type Colors = {
 
     // GLOBAL
     brand: string;
-    brandDark: string;
+    // brandHigh: string;
```
Comment (yceballost) [inline src/skins/types.tsx:97]: why this comment?

Comment (atabel) [reply src/skins/types.tsx:97]: leftovers, I'll remove it

File: src/skins/movistar.tsx [hunk 2]
```diff
@@ -139,7 +139,7 @@ export const getMovistarSkin: GetKnownSkin = (variant) => {
 
             // GLOBAL
             brand: palette.movistarBlue,
-            brandDark: palette.movistarBlue55,
+            brandHigh: palette.movistarBlue55,
             inverse: palette.white,
             neutralHigh: palette.grey6,
             neutralMedium: palette.grey5,
```
Comment (yceballost) [inline src/skins/movistar.tsx:145]: neutralLow wrong color, change to `grey1`
https://github.com/Telefonica/mistica-design/blob/f823f0765e6e8491e883a2e7019a736e1206a359/tokens/movistar.json#L299

File: src/skins/o2.tsx [hunk 3]
```diff
@@ -136,7 +136,7 @@ export const getO2Skin: GetKnownSkin = () => {
 
             // GLOBAL
             brand: palette.o2BluePrimary,
-            brandDark: palette.o2BluePrimary70,
+            brandHigh: palette.o2BluePrimary70,
             inverse: palette.white,
             neutralHigh: palette.grey6,
             neutralMedium: palette.grey5,
```
Comment (yceballost) [inline src/skins/o2.tsx:142]: neutralLow → `grey1`
https://github.com/Telefonica/mistica-design/blob/f823f0765e6e8491e883a2e7019a736e1206a359/tokens/o2.json#L298

File: src/skins/o2-classic.tsx [hunk 4]
```diff
@@ -146,7 +146,7 @@ export const getO2ClassicSkin: GetKnownSkin = () => {
 
             // GLOBAL
             brand: palette.o2Blue,
-            brandDark: palette.o2Blue55,
+            brandHigh: palette.o2Blue55,
             inverse: palette.white,
             neutralHigh: palette.grey6,
             neutralLow: palette.grey3,
```
Comment (yceballost) [inline src/skins/o2-classic.tsx:152]: neutralLow → `grey1`
https://github.com/Telefonica/mistica-design/blob/f823f0765e6e8491e883a2e7019a736e1206a359/tokens/classic-o2.json#L298

File: src/skins/vivo.tsx [hunk 5]
```diff
@@ -125,7 +125,7 @@ export const getVivoSkin: GetKnownSkin = () => {
 
             // GLOBAL
             brand: palette.vivoPurple,
-            brandDark: palette.vivoPurpleDark,
+            brandHigh: palette.vivoPurpleDark,
             inverse: palette.white,
             neutralHigh: palette.grey6,
             neutralMedium: palette.grey5,
```
Comment (yceballost) [inline src/skins/vivo.tsx:131]: neutralLow → `grey1`
https://github.com/Telefonica/mistica-design/blob/f823f0765e6e8491e883a2e7019a736e1206a359/tokens/vivo.json#L299

File: src/skins/blau.tsx [hunk 6]
```diff
@@ -133,7 +134,7 @@ export const getBlauSkin: GetKnownSkin = () => {
 
             // GLOBAL
             brand: palette.blauBluePrimary,
-            brandDark: palette.blauBlueSecondary,
+            brandHigh: palette.blauBlueSecondary,
             inverse: palette.white,
             neutralHigh: palette.grey6,
             neutralMedium: palette.grey5,
```
Comment (yceballost) [inline src/skins/blau.tsx:140]: neutralLow → `grey1`
https://github.com/Telefonica/mistica-design/blob/f823f0765e6e8491e883a2e7019a736e1206a359/tokens/vivo.json#L299

**Review by yceballost — CHANGES_REQUESTED** [review]

**Comment by yceballost** [comment]
I forgot to put in jira ticket that Empty State component also change the description to regular weight
image

**Review by pladaria — APPROVED** [review]

**Review by yceballost — APPROVED** [review]
superb! <3

**Comment by yceballost** [comment]
I found a bug between actual versions and this PR
image

https://ibit.ly/09Ja

Maybe is the same bug found in screenshots WEB-596

**Pull request merged**